game/src/js/scenes/DungeonScene.js

```diff
@@ -67,10 +68,28 @@ export default class DungeonScene extends BaseScene {
 
   create() {
     this.spawn(0,0);
-
+    this.delay = 50;
+    this.enemyList = [];
+    this.virtualTime = 0;
   }
 
   update() {
+    --this.delay;
+    ++this.virtualTime;
+    if (this.delay <= 0) {
+      var newMonster = new FireMonster(this, 250 * (Math.random() - 0.5), 250 * (Math.random() - 0.5));
+      newMonster.setAI(this.localCharacter);
+      this.enemyList.push(newMonster);
+      this.characters.add(newMonster);
```

Comment:
Looks like this is adding the character to this list properly, but no action is taken when a collision occurs. Could you add a collision handler? See line 24.